front-end/src/Components/Layouts/EditOrganization.tsx

```diff
@@ -0,0 +1,129 @@
+  };
+
+  return (
+    <Fragment>
+      <Dialog
+        fullScreen
+        open={currentOrganizationData !== null}
+        onClose={handleClose}
+      >
+        <AppBar className={classes.appBar}>
+          <Toolbar>
+            <IconButton
+              edge="start"
+              color="inherit"
+              onClick={handleClose}
+              aria-label="close"
+            >
+              <CloseIcon />
+            </IconButton>
+            <Typography variant="h6" className={classes.title}>
+              Edit organization
+            </Typography>
+          </Toolbar>
+        </AppBar>
+        <Container className={classes.pageTitle}>
+          <Typography variant="h3" id="profileTourStep1">
+            Profile
+          </Typography>
+        </Container>
+        <Container>
+          <Grid container spacing={6}>
+            <Grid item xs={12} sm={12} md={4} lg={3} id="profileTourStep10">
+              {/* <ViewEditOrganizationPhoto
+            currentOrganizationData={currentOrganizationData}
+            currentUserProfile={currentUserProfile}
+            forceReloadUserData={forceReloadUserData}
+            handleLoadUserData={handleLoadUserData}
+            setPageKey={setPageKey}
+            setLoadingMessage={setLoadingMessage}
+            setNotification={setNotification}
+            classes={classes}
+          /> */}
+            </Grid>
+            <Grid item xs={12} sm={12} md={8} lg={9}>
+              <ViewEditOrganizationName
+                currentOrganizationData={currentOrganizationData}
+                setNotification={setNotification}
+                handleLoadOrganizationData={handleLoadOrganizationData}
+                setLoadingMessage={setLoadingMessage}
+                classes={classes}
+              />
+              <ViewEditOrganizationDescription
+                currentOrganizationData={currentOrganizationData}
+                setNotification={setNotification}
+                handleLoadOrganizationData={handleLoadOrganizationData}
+                setLoadingMessage={setLoadingMessage}
+                classes={classes}
+              />
+              <ViewEditOrganizationEmail
+                currentOrganizationData={currentOrganizationData}
+                setNotification={setNotification}
+                handleLoadOrganizationData={handleLoadOrganizationData}
+                setLoadingMessage={setLoadingMessage}
+                classes={classes}
+              />
+              <ViewEditOrganizationPhone
+                currentOrganizationData={currentOrganizationData}
+                setNotification={setNotification}
+                handleLoadOrganizationData={handleLoadOrganizationData}
+                setLoadingMessage={setLoadingMessage}
+                classes={classes}
+              />
+            </Grid>
+          </Grid>
+        </Container>
+      </Dialog>
+    </Fragment>
+  );
+};
+
+export default Organization;
```

Comment:
Can we call this "EditOrganization" to match the file name?

Reply:
Done